### DISCUSSION TITLE
How to setting Persistent storage mailpit on windows laragon?

### DISCUSSION THREAD

**Discussion by bagushermawan** [leokhoa/laragon#631] [Q&A] [answered]
everytime i start, then stop, all email was reseted.
how do i prevent email from being reset?

**Comment by Pen-y-Fan** [comment, marked as answer]
See the mailpit documentation: https://mailpit.axllent.org/docs/configuration/email-storage/

**Reply by bagushermawan** [reply]
i type like this 'mailpit --db-file C:\laragon\bin\mailpit'

![image](https://github.com/leokhoa/laragon/assets/54248013/3208bb58-d852-4da4-96da-79b25376e476)

what is the correct command sir?

**Reply by bagushermawan** [reply]
oh that happen cause i already run in another cmd, fixed thanks sir
